src/main/java/org/openstreetmap/atlas/geography/sharding/preparation/TilePrinter.java

```diff
@@ -83,6 +83,7 @@ protected int onRun(final CommandMap command)
                     writer.write(";");
                 }
                 writer.write("\n");
+                tile.freeBoundsForGC();
```

Comment:
Do you mind adding a quick comment, similar to the PR description about why this is needed her.